doc/clang.txt

```diff
@@ -265,6 +265,7 @@ A: Please use with neocomplete, they can work well together.
         if !exists('g:neocomplete#force_omni_input_patterns')
           let g:neocomplete#force_omni_input_patterns = {}
         endif
+        let g:neocomplete#force_overwrite_completefunc = 1
```

Comment:
Hi, Shougo.

Very excited to get your note for this issue! Thanks for your vim plugins. They are really helpful to me. I am planing to figure out how neocomplete working with asynchronous (as it's too slow sometimes for large c++ project) complete sources like clang. I need neosnippet and clang complete simultaneously. Would you have any advice?

B.R.
maxint